.rubocop.yml

```diff
@@ -59,6 +59,7 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Exclude:
     - test/**/*.rb
+    - 'config/routes.rb'
```

Comment:
It was probably time we did this. Might be worth updating `Metrics/BlockLength` in `.rubocop_todo.yml` to a lower value, since we've just been incrementing it each time we updated `routes.rb`.

Reply:
> since we've just been incrementing it each time we updated routes.rb.

Oh haha TIL, will do 🫡

Reply:
Updated the rubocop files accordingly. Also added tasks and concerns to the mix because the content of a task or concern is supposed to be surrounded by a block.